### DISCUSSION TITLE
Average Price for EC2 deployment?

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3750] [Help / Questions] [answered]
Hello, is there any list or statistics on how much you would be paying for an EC2 deployment on average for certain conditions? Something like a success story somewhere?
Because I know EC2 is just not multiplying your tier * 24 hours * 30 days and that's it but there are other costs like network and database possibly right.

**Comment by NGPixel** [comment, marked as answer]
If you're using the AWS Marketplace image of Wiki.js, the database is included within the EC2 instance. The costs depends on:

- The EC2 instance type
- The attached disk (speed + size)
- The network transfer out

All of the above depends on the region and your usage. 

If you're not comfortable with these variable costs, consider using a host like DigitalOcean where the prices are flat and predictable.
